vault/resource_database_secret_backend_connection.go

```diff
@@ -1659,11 +1675,11 @@ func validateDBPluginName(s string) error {
 func getSortedPluginPrefixes() ([]string, error) {
 	var pluginPrefixes []string
 	for _, d := range dbEngines {
-		prefix, err := d.PluginPrefix()
+		prefix, err := d.PluginPrefixes()
```

Comment:
nit: should be plural
```suggestion
		prefixes, err := d.PluginPrefixes()
```